metricbeat/mb/registry.go

```diff
@@ -94,7 +115,28 @@ func (r *Register) AddModule(name string, factory ModuleFactory) error {
 // HostParser function for parsing the 'host' configuration data. An error is
 // returned if any parameter is empty or nil or if a factory has already been
 // registered under the name.
+//
+// Use MustAddMetricSet for new code.
```

Comment:
Does that mean we need to update all existing metricsets?

Reply:
UPDATE: Just read your comment again in the PR and does not seem to be the case.